textdescriptives/load_components.py

```diff
@@ -0,0 +1,19 @@
+"""Adds components to a spaCy pipeline"""
+from .components.utils import create_utils_component
+from .components.readability import create_readability_component
+from .components.dependency_distance import create_dependency_distance_component
+from .components.descriptive_stats import create_descriptive_stats_component
+
+from spacy.language import Language
+
+
+def add_components(nlp: Language):
+    """Add all components to a spaCy pipeline"""
+    for component in [
+        "utilities",
+        "descriptive_stats",
+        "readability",
+        "dependency_distance",
+    ]:
+        nlp.add_pipe(component, last=True)
+    return nlp
```

Comment:
Seems like you should just use ´nlp.add_pipe("textdescriptives", last=True)` instead and then make a common pipe. 

The alternatives should naturally work independently as well:
´nlp.add_pipe("descriptive_stats", first=True)`
´nlp.add_pipe("readability", first=True)`
´nlp.add_pipe("dependency_distance", last=True)`

Reply:
added in 43f3a5b